mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule Malomo.MixProject do
   def project do
     [
       app: :malomo,
-      version: "0.0.2",
-      elixir: "~> 1.7",
+      version: "0.0.3",
+      elixir: "~> 1.8",
```

Comment:
Don't bump the Elixir version number. This represents the _minimum_ supported Elixir version. Anyone running a version of Elixir lower than 1.8 will run into issues.